package.json

```diff
@@ -70,9 +70,10 @@
   "dependencies": {
     "@truffle/hdwallet-provider": "^1.0.0",
     "dotenv": "^8.0.0",
+    "global": "^4.4.0",
     "openzeppelin-solidity": "^2.0.0",
     "shelljs": "^0.8.3",
-    "solc": "0.5.17",
+    "solc": "0.7.1",
```

Comment:
might be nice to go to the latest `0.7.x` version

Reply:
Done.

Reply:
Done.